waku/v2/node/config.nim

```diff
@@ -83,8 +84,13 @@ type
 
     rlnRelayPubsubTopic* {.
       desc: "the pubsub topic for which rln-relay gets enabled",
-      defaultValue: "waku/2/rlnrelay/proto"
+      defaultValue: "/waku/2/default-waku/proto"
       name: "rln-relay-pubsub-topic" }: string
+
+    rlnRelayContentTopic* {.
+      desc: "the pubsub topic for which rln-relay gets enabled",
+      defaultValue: "waku/2/huilong/proto"
```

Comment:
I understand that we may want to default to the `huilong` content topic for `chat2`, but wouldn't it make sense for the default behaviour of a `wakunode2` to enable rln for "all" content topics (I think that corresponds to an empty string `""`)?
The `huilong` content topic is `/toy-chat/2/huilong/proto`

Reply:
> The huilong content topic is /toy-chat/2/huilong/proto.

Had it fixed locally but forgot to push, thanks. Please see https://github.com/status-im/nim-waku/pull/835/commits/44bef3c4d27e8a2befabeff6ebc6063f7812e014

Also added support for any contentTopic for a specific pubsub topic, please see https://github.com/status-im/nim-waku/pull/835/commits/10ecc77a86b46be6bf9ae9ded5dea93379fa489a